pages/create-post.tsx

```diff
@@ -0,0 +1,43 @@
+import CreatePostWizard from "@components/create-post-page/CreatePostWizard";
+import Layout from "@components/Layout";
+import LoginModal from "@components/LoginModal";
+import pageTitle from "@lib/helpers/frontend/page-title";
+import { useUserQuery } from "@lib/hooks/user";
+import type { NextPage } from "next";
+import Head from "next/head";
+
+const PAGE_TITLE = "Create Post";
+
+const CreatePosting: NextPage = () => {
+  const { user } = useUserQuery();
+
+  if (!user) {
+    return (
+      <>
+        <Head>
+          <title>{PAGE_TITLE}</title>
+        </Head>
+        <Layout>
+          <LoginModal
+            isOpen={true}
+            message="To create a post you must be signed in. Please login with your MacID below."
+          />
```

Comment:
Hm, maybe it's better to have an `onClose` prop that calls [`router.back()`](https://nextjs.org/docs/api-reference/next/router#routerback) in this case because if you click on the sell book button from the header without being logged in this traps you in the UI.